src/frontend/src/lib/derived/network-tokens.derived.ts

```diff
@@ -6,28 +6,33 @@ import { tokens, tokensToPin } from '$lib/derived/tokens.derived';
 import { balancesStore } from '$lib/stores/balances.store';
 import type { Token, TokenUi } from '$lib/types/token';
 import { filterTokensForSelectedNetwork } from '$lib/utils/network.utils';
-import { pinTokensWithBalanceAtTop, sortTokens } from '$lib/utils/tokens.utils';
+import {
+	filterEnabledTokens,
+	pinTokensWithBalanceAtTop,
+	sortTokens
+} from '$lib/utils/tokens.utils';
 import { derived, type Readable } from 'svelte/store';
 
+/**
+ * All enabled by user tokens.
+ */
+const enabledTokens: Readable<Token[]> = derived([tokens], filterEnabledTokens);
```

Comment:
Should this derived store find place in related `tokens.derived` module?

Reply:
Good point, moved.